### DISCUSSION TITLE
Not authenticated on minecraft.net

### DISCUSSION THREAD

**Discussion by 4rturoooo** [games647/FastLogin#757] [Q&A] [answered]
Hello, I have installed the installation guide to the letter, but I have two problems, first I tried to enter with tlauncher and it tells me that I am not authenticated, but I checked everything and I do not have the online mode set to true, and two (that this is in my premium account), it is that the "Invalid session" does not appear, it only appears when I have the /premium, and I want it to always appear if I have the session open for a long time as universecraft

**Comment by TuxCoding** [comment, marked as answer]
This error is client-side when the plugin requests an onlinemode connection like in vanilla. Your client decides to disconnect, because there is no valid session. Check for any errors in the client log.

You should also try it with the newest vanilla launcher without any mods.

**Comment by 4rturoooo** [comment]
Oh, okay, I've seen, it's my client's problem, but what about the second?

**Reply by TuxCoding** [reply]
You need to explain that one.
